crates/host/src/wasmbus/mod.rs

```diff
@@ -1766,31 +1766,7 @@ impl Host {
         } else {
             Arc::new(KeyPair::new(KeyPairType::Server))
         };
-
-        let mut labels = HashMap::from([
```

Comment:
You'll need to make sure you update places that were using `labels` to use `config.labels` instead in this function. You'll also need to remove some old imports you no longer need. If you run `cargo check --workspace`, you should see all of the compile errors and the warnings of imports you should remove